.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: python
+cache: pip
+
+env:
+  global:
+    - PYTHONPATH="./tests/"
+    - DJANGO_SETTINGS_MODULE="settings_test"
+
+matrix:
+  include:
+    - python: 3.6
+    # Workaround for Python 3.7 support. See:
+    # https://github.com/travis-ci/travis-ci/issues/9815
+    - python: 3.7
+      dist: xenial
+      sudo: true
+
+# Workaround for testing Boto3 on Travis. Taken from:
+# https://github.com/azavea/django-amazon-ses/blob/57ed8619435194742308c6f15cbf9c91b231b88a/.travis.yml#L17-L19
+before_install: sudo rm /etc/boto.cfg
+
+install: pip install .[tests]
+
+script: django-admin test --noinput
+
+deploy:
+  - provider: pypi
```

Comment:
I would really like to get this working under `azavea` before we starting trying to consume it in other apps. I don't remember the reason why things didn't work, but if they challenging, then I think we should open a separate issue for it.

Reply:
I was able to get this working. 

See this successful build: https://travis-ci.org/azavea/django-ecsmanage/jobs/517419624

This is a good comment that explains why it was challenging: https://travis-ci.community/t/deploy-to-pypi-fails-with-authentication-error/1964/7

😰